nbgitpuller/pull.py

```diff
@@ -304,13 +304,14 @@ def main():
     parser = argparse.ArgumentParser(description='Synchronizes a github repository with a local repository.')
     parser.add_argument('git_url', help='Url of the repo to sync')
     parser.add_argument('branch_name', default=None, help='Branch of repo to sync', nargs='?')
-    parser.add_argument('repo_dir', default='.', help='Path to clone repo under', nargs='?')
+    parser.add_argument('--target-dir', default='.', help='Path to clone repo under')
+
     args = parser.parse_args()
 
     for line in GitPuller(
         args.git_url,
-        args.repo_dir,
-        branch=args.branch_name if args.branch_name else None
+        args.target_dir,
+        branch=args.branch_name
```

Comment:
These show up again, not part of the documentation PR right?

Btw, I typically do a lot of the following to avoid accidental commits:
- `git status` - what is the situation?
- `git diff` - what is changed and unstaged?
- `git diff --staged` - what am I about to commit?
- `git diff HEAD~1..HEAD` - what did I commit in the last commit?

Then to correct mistakes I do:
- `git rebase -i HEAD~X` where X is some numbers of commits to modify up in the history, and I reorder commits, change titles, stop and edits sometimes, squash/fixup etc.
- `git commit --amend` or `git commit --amend --no-edit` to modify the latest commit etc

Reply:
yes, this is embarrassing!  thanks, I will pull it out. In my first moments of working on this project a bunch of months ago, I worked on the command-line args piece. I made the changes off the master branch of my fork not realizing this would cause me headaches if it didn't merge in quickly.

Reply:
I have a plenty of understanding with the situation @sean-morris, no worries! It is very complicated when any large or several changes are planned at the same time.